code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -120,10 +122,12 @@
 			cached_gases[/datum/gas/oxygen][MOLES] -= cached_gases[/datum/gas/tritium][MOLES]
 
 		if(burned_fuel)
-			energy_released += FIRE_CARBON_ENERGY_RELEASED * burned_fuel
+			energy_released += FIRE_HYDROGEN_ENERGY_RELEASED * burned_fuel
+			if(location)
+				radiation_pulse(location, energy_released/TRITIUM_BURN_RADIOACTIVITY_FACTOR)
```

Comment:
I don't mind merging the part where tritium burns into water, but the rad memes are a bit much considering the relatively inert nature of tritium and `radiation_pulse()` not being free performance-wise.

Reply:
But... fusion does (did) the exact same thing, and it was considered fine (it was disabled for other reasons). Not only that, but tritum burns REALLY quickly so you looking at a couple seconds of this at most.

It might still pulse when it's doing the weird endothermic burn thing, so if you'd like I could change it so it only radiates during the brief, O2-rich burn.

Reply:
@vuonojenmustaturska